cdap-docs/user-guide/source/data-preparation/functions/table-of-contents.rst

```diff
@@ -1,18 +0,0 @@
-.. meta::
```

Comment:
The TOC tree is included in the index for the subfolders (following structure left by JJ before), so including a table-of-contents will cause a circular reference error.